webapp/components/Map.tsx

```
@@ -1,27 +1,422 @@
+      coordinates[0] += e.lngLat.lng > coordinates[0] ? 360 : -360;
+    }
+
+    // Populate the popup and set its coordinates
+    // based on the feature found.
+    popup.setLngLat(coordinates).setHTML(description).addTo(_map);
+  });
+
+  _map.on("mouseleave", MAP_LAYER_COMMUNES_DOTS, () => {
+    _map.getCanvas().style.cursor = "";
+    popup.remove();
+  });
```

Biome (1.9.4)

[error] This code is unreachable

... because this statement will return from the function beforehand


(lint/correctness/noUnreachable)